test/dd/test_package.cpp

```diff
@@ -50,8 +49,8 @@ TEST(DDPackageTest, TrivialTest) {
   auto dd = std::make_unique<dd::Package<>>(2);
   EXPECT_EQ(dd->qubits(), 2);
 
-  auto xGate = dd->makeGateDD(dd::X_MAT, 0);
-  auto hGate = dd->makeGateDD(dd::H_MAT, 0);
+  auto xGate = dd->makeGateDD(dd::opToSingleGateMatrix(qc::X), 0);
```

Comment:
Although these are just tests, this might be a nice opportunity to modernise the syntax in some of these.
Instead of directly calling `makeGateDD` here one could construct a `StandardOperation` and call the `getDD` method. Feels slightly less clunky and manual.